compute/test/createComputeHyperdiskFromPool.test.js

```diff
@@ -18,14 +18,14 @@
 
 const path = require('path');
 const {assert} = require('chai');
-const {after, before, describe, it} = require('mocha');
+const {after, before, xdescribe, it} = require('mocha');
```

Comment:
Skip until https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3808 is not resolved